spec-main/extensions-spec.ts

```diff
@@ -147,6 +147,21 @@ ifdescribe(process.electronBinding('features').isExtensionsEnabled())('chrome ex
       expect(response).to.equal(3)
     })
 
+    it('connect', async () => {
+      const customSession = session.fromPartition(`persist:${require('uuid').v4()}`)
+      await customSession.loadExtension(path.join(fixtures, 'extensions', 'chrome-api'))
+      const w = new BrowserWindow({ show: false, webPreferences: { session: customSession, nodeIntegration: true } })
+      await w.loadURL(url)
+
+      const portName = require('uuid').v4()
+      const message = { method: 'connectTab', args: [portName] }
+      w.webContents.executeJavaScript(`window.postMessage('${JSON.stringify(message)}', '*')`)
+
+      const [,, responseString] = await emittedOnce(w.webContents, 'console-message')
+
+      expect(responseString).to.equal(portName)
```

Comment:
Does this test that the resulting port works (i.e. you can send messages across it)?

Reply:
It only tests that the port was created, good catch! I'll send a message through to verify.